File: docs/vsto/codesnippet/CSharp/Trin_TaskPaneMonthCalendar/MyUserControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;

//<Snippet1>
using Office = Microsoft.Office.Core;
using PowerPoint = Microsoft.Office.Interop.PowerPoint;
//</Snippet1>

namespace Trin_TaskPaneMonthCalendar
{
    public partial class MyUserControl : UserControl
    {
        //<Snippet2>
        private PowerPoint.Shape textbox;
        //</Snippet2>

        public MyUserControl()
        {
            InitializeComponent();
        }

        //<Snippet3>
        private void monthCalendar1_DateChanged(object sender, DateRangeEventArgs e)
        {
            try
            {
                if (textbox != null)
                {
                    textbox.Delete();
                }

                PowerPoint.Slide slide =
                    Globals.ThisAddIn.Application.ActivePresentation.Slides[1];
                textbox = slide.Shapes.AddTextbox(
                    Office.MsoTextOrientation.msoTextOrientationHorizontal,
                    50, 100, 600, 50);
                textbox.TextFrame.TextRange.Text = e.Start.ToLongDateString();
                textbox.TextFrame.TextRange.Font.Size = 48;
                textbox.TextFrame.TextRange.Font.Color.RGB =
                     Color.DarkViolet.ToArgb();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.ToString());
            }
        }
        //</Snippet3>
    }
}

File: docs/vsto/codesnippet/VisualBasic/Trin_TaskPaneMonthCalendar/MyUserControl.vb

'<Snippet1>
Imports Office = Microsoft.Office.Core
Imports PowerPoint = Microsoft.Office.Interop.PowerPoint
'</Snippet1>

Public Class MyUserControl

    '<Snippet2>
    Private TextBox As PowerPoint.Shape
    '</Snippet2>

    '<Snippet3>
    Private Sub MonthCalendar1_DateChanged( _
        ByVal sender As System.Object, _
        ByVal e As System.Windows.Forms.DateRangeEventArgs) _
        Handles MonthCalendar1.DateChanged

        Try
            If TextBox IsNot Nothing Then
                TextBox.Delete()
            End If

            Dim Slide As PowerPoint.Slide
            Slide = Globals.ThisAddIn.Application.ActivePresentation.Slides(1)
            TextBox = Slide.Shapes.AddTextbox( _
                Office.MsoTextOrientation.msoTextOrientationHorizontal, _
                50, 100, 600, 50)
            TextBox.TextFrame.TextRange.Text = e.Start.ToLongDateString()
            TextBox.TextFrame.TextRange.Font.Size = 48
            TextBox.TextFrame.TextRange.Font.Color.RGB = _
                System.Drawing.Color.DarkViolet.ToArgb()

        Catch ex As Exception
            System.Windows.Forms.MessageBox.Show(ex.ToString())
        End Try
    End Sub
    '</Snippet3>
End Class
